splinepy/microstructure/tiles/cube_void.py

```
@@ -25,6 +25,17 @@ class CubeVoid(_TileBase):
     _para_dim = 3
     _evaluation_points = _np.array([[0.5, 0.5, 0.5]])
     _n_info_per_eval_point = 4
+    _sensitivities_implemented = True
+    # TODO: clever parameter bounds and checks if given parametrization would
+    # still lie in unit cube
+    _parameter_bounds = [
+        [0.0, 1.0],
+        [0.0, 1.0],
+        [-_np.pi / 2, _np.pi / 2],
+        [-_np.pi / 2, _np.pi / 2],
+    ]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)